guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -132,64 +133,39 @@ ID | DESTINATION SERVER | PATH
 You can use the archive that the `hammer content-export` command outputs to create a version of a Content View with the same content as the exported Content View version.
 For more information about exporting a Content View version, see xref:Using_ISS-Exporting-a-Content-View-Version[].
 
-When you import a Content View version, it has the same major and minor version numbers and contains the same repositories with the same packages and errata.
-You can customize the version numbers by changing the `major` and `minor` settings in the `json` file that is located in the exported archive.
+When you import a Content View version, it has the same major and minor version numbers and contains the same repositories with the same packages and errata. The Custom Repositories, Products and Content Views are automatically created if they don't exist in the importing organization.
+
 
 .Prerequisites
 
 To import a Content View, ensure that {ProjectServer} where you want to import meets the following conditions:
 
-ifdef::satellite[]
-* If you want to import a Content View to {Project} in a disconnected environment, you must configure {Project} to synchronize content with a local CDN server and then synchronize content that the CV you export contains.
-For more information, see xref:configuring-satellite-to-synchronize-content-with-a-local-cdn-server_content-management[].
-endif::[]
-
-* Ensure that you set download policy to *Immediate* for all repositories within the Content View you export.
-
-For more information, see xref:Managing_Content_Views-Creating_a_Simple_Content_View[].
+. The exported dump must be in a directory under `/var/lib/pulp/imports` . More over the directory must have `pulp:pulp` permissions for pulp to be able to read and write the json files in that directory.
```

Comment:
```suggestion
. The exported dump must be in a directory under `/var/lib/pulp/imports`. 
. The directory must have `pulp:pulp` permissions so that Pulp can read and write the `.json` files in that directory.
```